docs/pages/kubernetes-access/helm/guides/gcp.mdx

```diff
@@ -0,0 +1,422 @@
+---
+title: Running an HA Teleport cluster using GCP, GKE and Helm
+description: Install and configure an HA Teleport cluster using a Google Cloud GKE cluster
+---
+
+In this guide, we'll go through how to set up a highly-available Teleport cluster with multiple replicas in Kubernetes
+using Teleport Helm charts and Google Cloud Platform products (Firestore and Google Cloud Storage).
+
+### Install Helm
+
+Teleport's charts require the use of Helm version 3. You can [install Helm 3 by following these instructions](https://helm.sh/docs/intro/install/).
+
+Throughout this guide, we will assume that you have the `helm` and `kubectl` binaries available in your `PATH`:
+
+```bash
+$ helm version
+version.BuildInfo{Version:"v3.4.2"}
```

Comment:
Let's make this a variable in config.json and refer to it  as `(=helm.version=)`, same re: kubectl version